app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,44 @@
+class FriendshipsController < ApplicationController
+  def index
+    @user = User.find_by(params[:user_id])
+    @friendships = @user.friendships
+  end
+
+  def update
+    @user = User.find_by(params[:user_id])
+    @friendship = @user.friendships.find params[:id]
+    @friendship.accept
+  end
+
```

Comment:
From your logic and inspection of your model, you should create the inverse friendship from here. You can generate a method on your model to do this and create a call back that creates the inverse friendship. 
What this means is once confirmed for a friendship is "true", create another friendship by reversing the id's for user and friend and both entries should be true.
e.g user_id: 1 friend_id: 2 confirmed: true <=====> user_id: 2 friend_id:1 confirmed: true.